pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -421,6 +421,17 @@ type awsElasticBlockStoreProvisioner struct {
 var _ volume.Provisioner = &awsElasticBlockStoreProvisioner{}
 
 func (c *awsElasticBlockStoreProvisioner) Provision() (*v1.PersistentVolume, error) {
+	var FSType string = "ext4"
```

Comment:
I think we prefer lower case first char for local variables `var fsType string = ext4`

You can also do `fsType := "ext4"`

(and I think this needs to be done for every provider)